modules/determining-upgrade-viability-cv-conditiontype.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * updating/index.adoc
+
+:_content-type: CONCEPT
+[id="Understanding_clusterversion_conditiontypes_{context}"]
+= Understanding cluster version condition types
+ 
+The status of the cluster version and its Operators include the condition type, informing you of the health and current state of your cluster version and Operator.
+
+In addition to `Available`, `Progressing`, and `Upgradeable`, there are additional condition types that affect cluster versions and Operators.
+
+The Cluster Version Operator (CVO) is responsible for collecting the status conditions from cluster Operators so that cluster administrators can better understand the state of the {product-title} cluster.
+
+* Failing: 
+The CVO monitors cluster Operators and other components. `ClusterStatusFailing` is set on the cluster version status when a cluster cannot reach the desired state. It is considered more severe than `Degraded`, indicates that the cluster is not healthy, and requires an administrator to intervene.
```

Comment:
same as Yang's comment from below. `ClusterStatusFailing` is the name of the variable in source code, however `Failing` is the actual condition being exposed to the user via both `oc adm upgrade` command and via `.status.conditions`

Reply:
as for if the following statement is true:
> It is considered more severe than `Degraded`

maybe @wking have an opinion on that

Reply:
and if I recall it right, `Failing` is actually used **instead** of `Degraded` in CVO, so the aforementioned statement can be wrong in another way too

Reply:
Updated to Failing :) ty! @wking @LalatenduMohanty any thoughts on "It is considered more severe than `Degraded`, indicates that the cluster is not healthy, and requires an administrator to intervene." ?

Reply:
Removed :)